docs/docs/migrating-from-v1-to-v2.md

```diff
@@ -412,8 +412,8 @@ const NewsFeed = () => (
 )
 
 const Photo = ({ location, photoId }) => {
-  if (location.state.fromFeed) {
-    return <FromFeedPhoto id={photoId} />
+  if (location.state.fromNewsFeed) {
```

Comment:
Why was this changed?

Reply:
Because the example shows that <Link /> in NewsFeed has `fromNewsFeed` state but the props that Photo received is `fromFeed`

```jsx
const NewsFeed = () => (
  <div>
    <Link to="photos/123" state={{ fromNewsFeed: true }} />
  </div>
)

const Photo = ({ location, photoId }) => {
  if (location.state.fromFeed) {
    return <FromFeedPhoto id={photoId} />
  } else {
    return <Photo id={photoId} />
  }
}
```